SS_readdata_330.tpl

```diff
@@ -202,18 +202,19 @@
   }
  END_CALCS
 
-//  SS_Label_Info_2.1.5  #Define fleets, surveys and areas
+//  SS_Label_Info_2.1.5  #Define fleets, surveys, predators and areas
   imatrix pfleetname(1,Nfleet,1,2)
   ivector fleet_type(1,Nfleet)   // 1=fleet with catch; 2=discard only fleet with F; 3=survey(ignore catch); 4=M2=predator
   int N_bycatch;  //  number of bycatch only fleets
   int N_pred;  //  number of predator fleets
   ivector N_catchfleets(0,pop); //  number of bycatch plus landed catch fleets by area
   imatrix fish_fleet_area(0,pop,0,Nfleet)   // list of catch_fleets that are type 1 or 2, so have a F
-  ivector predator(1,Nfleet)   // list of "fleets" that are type 4, so are added to M rather than to F
-  ivector predator_rev(1,Nfleet)   // predator given f
+  ivector predator(1,Nfleet)   // list of "fleets" that are type 4
```

Comment:
style: add space `(1, Nfleet)` I think? Missing `;` at end of line?

Also needed on the two following lines.